monitor-manage-device-with-iotcentral/CoffeeMaker.js

```diff
@@ -0,0 +1,266 @@
+// Use the Azure IoT device SDK for devices that connect to Azure IoT Central.
+var iotHubTransport = require('azure-iot-device-mqtt').Mqtt;
+var Client = require('azure-iot-device').Client;
+var Message = require('azure-iot-device').Message;
+var ProvisioningTransport = require('azure-iot-provisioning-device-mqtt').Mqtt;
+var SymmetricKeySecurityClient = require('azure-iot-security-symmetric-key').SymmetricKeySecurityClient;
+var ProvisioningDeviceClient = require('azure-iot-provisioning-device').ProvisioningDeviceClient;
+
+// DPS connection information
+var modelIdObject = { modelId: 'dtmi:com:example:ConnectedCoffeeMaker;1' };
+var provisioningHost = 'global.azure-devices-provisioning.net';
+var idScope = '{ID scope}';
+var registrationId = '{Device ID}';
+var symmetricKey = '{Primary key}';
```

Comment:
I think this will make the error handling nicer and easier to understand if the user fails to provide this
```suggestion
var idScope = null; // put your ID scope here
var registrationId = null'; // put your Device ID here
var symmetricKey = null; // put your Primary key here
```